modules/integration/src/test/scala/scala/cli/integration/ExportMillTestDefinitions.scala

```diff
@@ -96,4 +96,9 @@ abstract class ExportMillTestDefinitions extends ScalaCliSuite
     test("JVM with compiler plugin") {
       jvmTestCompilerPlugin()
     }
+
+  test("Scala Native") {
+    // FIXME this should be adjusted to Scala Native 0.5.x syntax once Mill gets support for it
```

Comment:
This should have a follow up issue as the ones below.

Reply:
If you create an issue I will remember :P I snooze things into the future in my inbox :D 

Alternatively, we could have a label for those and have a bot update a comment etc. after a while 

We have something similar in Metals https://github.com/scalameta/metals/blob/main/.github/workflows/close-inactive-needs-more-info-issues.yml

Reply:
https://github.com/VirtusLab/scala-cli/issues/2870
I created an umbrella ticket.